sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/client/DefaultHttpClient.java

```diff
@@ -46,6 +48,13 @@ class DefaultHttpClient implements HttpClient {
     private final long connectionTimeout;
     private final long readTimeout;
     private final ProxyOptions proxyOptions;
+    private EventStreamListener listener;
+    private static final String DEFAULT_EVENT = "message";
+    private final Pattern DIGITS_ONLY = Pattern.compile("^[\\d]+$");
+    protected long lastEventID = -1;
+    protected long retryAfter;
```

Comment:
It is to acoomodate the field "retry" from the SSE spec 

>If the field name is "retry"
If the field value consists of only [ASCII digits](https://infra.spec.whatwg.org/#ascii-digit), then interpret the field value as an integer in base ten, and set the event stream's [reconnection time](https://html.spec.whatwg.org/multipage/server-sent-events.html#concept-event-stream-reconnection-time) to that integer. Otherwise, ignore the field.

https://html.spec.whatwg.org/multipage/server-sent-events.html